.eslintrc

```diff
@@ -10,15 +10,20 @@
     "react/prefer-es6-class": 0,
     "react/prefer-stateless-function": 0,
     "react/no-multi-comp": 0,
-    "array-callback-return": "off", 
+    "array-callback-return": "off",
     "arrow-body-style": ["warn", "as-needed"],
-    "comma-dangle": ["warn", "never"],
+    "comma-dangle": ["warn", "always"],
     "import/extensions": "off",
-    "import/no-unresolved": "off",
+    "import/no-unresolved": 1,
+    "indent": ["warn", 2],
     "react/jsx-no-bind": "off",
-    "max-len": "off",
+    "max-len": 1,
     "multiline-ternary": ["warn", "never"],
-    "no-else-return": "off",
+    "no-else-return": 1,
```

Comment:
I turned it on based on [a comment of yours else where](https://github.com/zooniverse/pandora/pull/4/files/42b46a4289376c997e07988e1f449d8749c4774b#r119328415), but happy to switch it off.